ner_v2/detectors/numeral/constant.py

```diff
@@ -0,0 +1,52 @@
+# Number detector constants
+
+NUMBER_DIGIT_UNITS = [0, 1, 1.5, 2, 2.5, 3, 4, 5, 6, 7, 8, 9]
+DIGIT_UNITS = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
+NUMERIC_VARIANTS = DIGIT_UNITS + [
+    "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen",
+    "sixteen", "seventeen", "eighteen", "nineteen", "and",
+    "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety",
+    "hundred", "thousand", "million", "billion", "trillion", "fourty", "ninty"
+]
+NUMBER_DATA_FILE_NUMBER = 'number'
+NUMBER_DATA_FILE_NAME_VARIANTS = 'name_variants'
+NUMBER_DATA_FILE_VALUE = 'value'
+NUMBER_DATA_FILE_TYPE = 'type'
+NUMBER_TYPE_UNIT = 'unit'
+NUMBER_TYPE_SCALE = 'scale'
+
+NUMBER_NUMERAL_CONSTANT_FILE = 'numerals_constant.csv'
+NUMBER_UNITS_FILE = 'units.csv'
```

Comment:
Put proper comments in constant file. For example 'units.csv', 'unit_variants', 'unit_value' should be together with one line comment for all three of them.

Reply:
Similarly 'numeral_constants.csv', 'scale', 'units', 'number', 'value', 'type' an be grouped together. Please do better renaming as 'value', 'type', 'unit' are getting used at multiple places and can get confusing.

Reply:
Rename value to ```# Numeral data file name and their columns
NUMBER_NUMERAL_CONSTANT_FILE_NAME = 'numerals_constant.csv'
NUMBER_NUMERAL_FILE_VARIANTS_COLUMN_NAME = 'name_variants'
NUMBER_NUMERAL_FILE_VALUE_COLUMN_NAME = 'number_value'
NUMBER_NUMERAL_FILE_TYPE_COLUMN_NAME = 'number_type'

# type value of number in numeral_constant data file
NUMBER_TYPE_UNIT = 'unit'


# Units data file and their columns
NUMBER_UNITS_FILE_NAME = 'units.csv'
NUMBER_DATA_FILE_UNIT_VARIANTS_COLUMN_NAME = 'unit_variants'
NUMBER_DATA_FILE_UNIT_VALUE_COLUMN_NAME = 'unit_value'

# Return type keys of number detection
NUMBER_DETECTION_RETURN_DICT_VALUE = 'value'
NUMBER_DETECTION_RETURN_DICT_UNIT = 'unit'```